app/views/layouts/_nav.html.erb

```diff
@@ -1,8 +1,7 @@
 <div class="prx-header-bar | navbar navbar-dark align-items-stretch p-0 bg-primary-dark">
   <div class="container-fluid align-items-stretch pe-0">
     <header class="flex-grow-1">
-      <h1 class="visually-hidden">PRX Dovetail</h1>
-      <%= link_to image_tag("dovetail_logo.svg", width: 152, height: 40, alt: "Dovetail"), main_app.podcasts_path, class: "navbar-brand" %>
+      <%= link_to image_tag("dovetail_logo.svg", width: 152, height: 30, alt: "Dovetail from PRX"), main_app.podcasts_path, class: "navbar-brand d-flex mt-2 mb-2" %>
```

Comment:
Added a little margin to the logo, its spaced better now. Hopefully I did this right by using the bootstrap class utility. Correct me if not. 

I'm learning!